contentcuration/contentcuration/frontend/accounts/pages/Main.vue

```diff
@@ -186,6 +186,16 @@
         }
         return Promise.resolve();
       },
+    },
+      if(
+        to.query?.showPolicy &&
+        Object.values(policies).includes(to.query.showPolicy) &&
+        !from.name   // Prevents the modal from showing when navigating back
```

Comment:
Any simpler/better conditions here?

The issue suggests this is particularly to accommodate users who have not signed up yet. I considered maybe using a route name (ie, is this only on the account sign in page?)

Reply:
The simpler solution would probably involve a bit more of a rewrite of the state - and just have the policy query parameter be the thing that determines whether the modal is shown or not.

Then closing the policy modal would just mean removing the query parameter, so you shouldn't get into this situation where you need to check the from route etc.